application/run-cron.sh

```diff
@@ -14,7 +14,7 @@ chown -R www-data:www-data \
   /data/console/runtime/
 
 # Run database migrations
-runny /data/yii migrate --interactive=0
+/data/yii migrate --interactive=0
```

Comment:
I forgot that one other feature of `runny` is that it causes the script to exit for non-zero return codes, rather than keep going. We'll probably want something here to cause the script to exit if the migrations fail now.

Reply:
I was wondering if I should add the `set -e`. Sounds like that would be important.

Reply:
Yep. I just wasn't sure what a `set -e` would do here in this run-cron.sh script, with its user of `term_handler()` and such.

Reply:
Good question. But FWIW, the `term_handler` is broken now that apache runs in the foreground, which happened when we moved from Logentries to Cloudwatch Logs. I haven't figured out a new solution for that yet.